sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2ScanRelationPushDown.scala

```diff
@@ -222,7 +231,7 @@ object V2ScanRelationPushDown extends Rule[LogicalPlan] with PredicateHelper {
       Cast(aggAttribute, aggDataType)
     }
 
-  def applyColumnPruning(plan: LogicalPlan): LogicalPlan = plan.transform {
+  def pushDownColumn(plan: LogicalPlan): LogicalPlan = plan.transform {
```

Comment:
maybe `pruneColumns`?

Reply:
I also think `pruneColumns` or the original name `applyColumnPruning` might be better.

Reply:
`pruneColumns` sounds good to me, thanks.